### PR TITLE
10 13 2023  as mc afee solidcore

### PR DESCRIPTION
**Pull request by AcceleryntSecurityDev**
Required items, please complete
   
   Change(s):
   - New Parser

   Reason for Change(s):
   - New Parser

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @AcceleryntSecurityDev, thanks for raising this PR, we will review it by 18 Oct 2023.

**Comment by AcceleryntSecurityDev** [comment]
@v-rbajaj thanks for the update!

**Comment by v-rbajaj** [comment]
Hi @AcceleryntSecurityDev, you need to convert this parser into yaml format. 
Please refer this below parser for the same.
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/AtlassianConfluenceAudit/Parsers/ConfluenceAudit.yaml

**Comment by AcceleryntSecurityDev** [comment]
@v-rbajaj hanks, I have added the yaml file

File: Parsers/AS-McAfeeSolidcore/AS-McAfeeSolidcore.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Function:
```
Comment (v-rbajaj) [inline Parsers/AS-McAfeeSolidcore/AS-McAfeeSolidcore.yaml:1]: Hi @AcceleryntSecurityDev, the `id` is missing, can you please add that?

Also can I please know why have you kept the txt, any particular reason, if not, please remove it, thanks

Comment (AcceleryntSecurityDev) [reply Parsers/AS-McAfeeSolidcore/AS-McAfeeSolidcore.yaml:1]: @v-rbajaj what value should I populate the id with? I will remove the txt file now. Thanks

Comment (v-rbajaj) [reply Parsers/AS-McAfeeSolidcore/AS-McAfeeSolidcore.yaml:1]: You need to add GUID - you can generate the GUID from here - https://guidgenerator.com/

**Comment by AcceleryntSecurityDev** [comment]
@v-rbajaj id has been added. Thanks!

**Review by v-rbajaj — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**